### ISSUE TITLE
Raw Raster HTML Broken

### ISSUE THREAD

**Issue by tresf** [qzind/tray#675]
Since the the merge of #625, raw raster html is broken.

Steps to reproduce:

1. Open sample.html
2. From the raw tab try the "HTML" button in the "Raster Print" section
   > <img src="https://user-images.githubusercontent.com/6345473/84928816-fe890200-b09c-11ea-9728-3c74f5d7d743.png" width="25%"/>

3. Observe the logs, a `1x1` image is printed
    >```
    >[INFO] 2020-06-17 13:19:06,983 @ qz.printer.ImageWrapper:95
    >    Loading BufferedImage
    >[INFO] 2020-06-17 13:19:06,983 @ qz.printer.ImageWrapper:96
    >    Dimensions: 1x1
    >```

The problem seems to be related to the zoom:

```java
pageZoom = model.getZoom();
System.out.println(pageZoom); // Prints "NaN"
```

The following patch produces an image, albeit not the right quality:
```diff
  pageZoom = model.getZoom();
+ if(Double.isNaN(pageZoom)) {
+     pageZoom = 1;
+ }
```

**Comment by tresf** [comment]
A temporary workaround is to force the undocumented zoom parameter per `zoom: 1`:

```diff
var data = [{
    type: 'raw',
    format: 'html',
    flavor: 'file',
    data: 'path/to/sample.html',
    // rendered pageWidth and pageHeight in pixels is required
-   options: { language: "ESCPOS", pageWidth: 500 },
+   options: { language: "ESCPOS", pageWidth: 500,  zoom: 1 },
}];
```

**Comment by tresf** [comment]
Preliminary patch available: 4568f56dacbe4a26c7107bc98b6c36848e8349e2

**Comment by tresf** [comment]
@bberenz the above patch with a `pageWidth` of `55` results in the following on Linux:

```
java.lang.RuntimeException: Requested texture dimensions (55x34466) require dimensions (55x0) that exceed maximum texture size (16384)
```
